### PR TITLE
Create it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Language list must be in alphabetical order
 pl
+it
```
Comment (whot) [inline po/LINGUAS:2]: hehe, did you see the comment above? :smile:

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,209 @@
+# Polish translation for tuhi.
```
Comment (whot) [inline po/it.po]: I admit my polish is at zero but my italian is slightly above zero, so I can say with some confidence that this file is not a polish translation :)

File: po/it.po [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+#. Translators: Do NOT translate or transliterate this text (this is an icon file name)!
+#: data/org.freedesktop.Tuhi.desktop.in:7
+msgid "org.freedesktop.Tuhi"
+msgstr "org.freedesktop.Tuhi"
+
+#. Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+#: data/org.freedesktop.Tuhi.desktop.in:12
+msgid "tablet;wacom;ink;"
+msgstr "tablet;wacom;ink;"
```
Comment (whot) [inline po/it.po]: shouldn't the search terms be translated too? or would you actually search for "ink" as opposed to "inchiostro" [1]


[1] google translate

File: po/it.po [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+msgid "Undo delete drawing"
+msgstr "Annulla l'eliminazione del disegno"
+
+#: data/ui/DrawingPerspective.ui:132
+msgid "Press the button on the device to synchronize drawings"
+msgstr "Premere il pulsante sul dispositivo per sincronizzare i disegni"
+
+#: data/ui/DrawingPerspective.ui:143 data/ui/SetupPerspective.ui:121
+msgid "page0"
+msgstr "pagina0"
```
Comment (whot) [inline po/it.po]: FTR, I filed #264 for this

**Review by whot — CHANGES_REQUESTED** [review]
Just a few nitpicks, please squash the two commits together as well, thanks.

File: po/it.po (hunk 4 above)
Comment (whot) [reply po/it.po]: #264 is merged now, if you rebase on top of master the various `page0` ones should go away.

**Comment by whot** [comment]
ftr, I can see the pushes you do but please ping me when you're ready for a review again so I don't waste time on an intermediate solution. Any comment you addressed please click the "Resolve conversation" button where there is nothing more to add, thanks.

**Comment by albanobattistella** [comment]
> ftr, I can see the pushes you do but please ping me when you're ready for a review again so I don't waste time on an intermediate solution. Any comment you addressed please click the "Resolve conversation" button where there is nothing more to add, thanks.

Hi, 
no other to add.

File: po/it.po [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+"Tuhi è un'interfaccia utente grafica per scaricare disegni archiviati su tablet "
+"dispositivi della gamma Wacom, ad esempio Intuos Pro Paper o Bamboo Slate."
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:15
+msgid ""
+"Tuhi requires Tuhi, the daemon to actually communicate with the devices."
+"ThiGui is merely a front end to Tuhi, Tuhi must be installed and running "
+"when Tuhi is launched."
+msgstr ""
+Tuhi richiede Tuhi, il demone per comunicare effettivamente con i dispositivi."
```
Comment (whot) [inline po/it.po]: i think there's a quote missing here

**Review by whot — CHANGES_REQUESTED** [review]
minor issue that I can spot here, let me know if that comment is bogus though

File: po/it.po (hunk 5 above)
Comment (albanobattistella) [reply po/it.po]: > i think there's a quote missing here

solved.

**Review by whot — APPROVED** [review]

**Comment by whot** [comment]
Thanks, merged

**Pull request merged**